portal/templates/scheduled_jobs_list.html

```diff
@@ -1,69 +1,48 @@
 {% extends "layout.html" %}
 {% block main %}
+    <h1 class="tnth-headline">Scheduled Jobs</h1>
 
-
-
-    <h1 class="tnth-headline">{{ _("Scheduled Jobs") }}</h1>
-
-    <h4>{{ _("Current Jobs") }}</h4>
+    <h4>Current Jobs</h4>
     {% if jobs %}
     <ul>
         {% for job in jobs %}
             <li>
-                <button title="Run Job" onclick="triggerJob(event, {{ job.id }})"><i class="fa fa-play"></i></button>
-                <b>{{ job.name }}</b>
-                <br />{{ _("Task") }}: {{ job.task }}
-                {% if job.args %}
-                <br />{{ _("Args") }}: {{ job.args }}
-                {% endif %}
-                {% if job.kwargs %}
-                <br />{{ _("Kwargs") }}: {{ job.kwargs }}
-                {% endif %}
-                <br />{{ _("Schedule") }}: {{ job.schedule }}
-                <br />{{ _("Active") }}: {{ job.active }}
-                <br />{{ _("Last Runtime") }}: {{ job.last_runtime }}
-                <br />{{ _("Last Status") }}: {{ job.last_status }}
-                <div id="triggerMessage_{{job.id}}" class="text-info"></div>
+                <div id="job_{{ job.id }}">
+                    <button title="Toggle Job" onclick="toggleJob(event, {{ job.id }})"><i id="activeIcon_{{job.id}}" class="{{'fa fa-toggle-on' if job.active else 'fa fa-toggle-off'}}" /></i></button>
+                    <button title="Run Job" onclick="triggerJob(event, {{ job.id }})"><i class="fa fa-play"></i></button>
+                    <b>{{ job.name }}</b>
+                    <div id="activeText_{{job.id}}" class="{{'text-info' if job.active else 'text-danger'}}">{{ "Active" if job.active else "Inactive" }}</div>
+                    Task: {{ job.task }}
+                    {% if job.args %}
+                    <br />Args: {{ job.args }}
+                    {% endif %}
+                    {% if job.kwargs %}
+                    <br />Kwargs: {{ job.kwargs }}
+                    {% endif %}
+                    <br />Schedule: {{ job.schedule }}
+                    <br />Last Runtime:
+                    <div id="lastRuntime_{{job.id}}" class="text-info">{{ job.last_runtime }}</div>
+                    Last Status:
+                    <div id="lastStatus_{{job.id}}" class="text-info">{{ job.last_status }}</div>
+                    <br />
+                </div>
+                <script src="{{ url_for('static', filename='js/scheduledJobs.js') }}"></script>
```

Comment:
@OptimusRhine Adam, looks great, thank you very much for the changes.  One thing I noticed here, I think the inclusion of script here should be moved out of the for loop otherwise it will be included multiple times.

Reply:
Ah, good catch :). Should be fixed now.